packages/ckeditor5-table/src/tableconfig.ts

```diff
@@ -271,6 +272,7 @@ export interface TableCellPropertiesConfig {
 	borderColors?: TableColorConfig;
 	backgroundColors?: TableColorConfig;
 	defaultProperties?: TableCellPropertiesOptions;
+	colorPicker?: false | ColorPickerConfig;
```

Comment:
I think we should mention this config somewhere in the doclets of `TableConfig` (`tableCellProperties?: TableCellPropertiesConfig;`)